artifactory_cleanup/rules/delete.py

```diff
@@ -102,3 +103,23 @@ def filter(self, artifacts):
         repositories = utils.build_repositories(artifacts)
         folders = utils.get_empty_folders(repositories)
         return folders
+
+
+class DeleteByRegex(Rule):
```

Comment:
Could you add some documentation for the rule in readme.md please? So we one can see how to use it